9444 push images (#9447)
IQSS/dataverse: src/main/docker/README.md

# Dataverse Application Container Image

> The "application image" offers you a deployment ready Dataverse application running on the underlying

Reviewer: ```suggestion
The "application image" offers you a deployment-ready Dataverse application running on the underlying
```

Author: Good catch, done.

application server, which is provided by the [base image](https://hub.docker.com/r/gdcc/base). 
Its sole purpose is to bundle the application and any additional material necessary to successfully jumpstart
the application.

Note: Until all :ref:`jvm-options` are *MicroProfile Config* enabled, it also adds the necessary scripting glue to
configure the applications domain during booting the application server. See :ref:`app-tunables`.

## Quick Reference

**Maintained by:**

This image is created, maintained and supported by the Dataverse community on a best-effort basis.

**Where to find documentation:**

The [Dataverse Container Guide - Application Image](https://guides.dataverse.org/en/latest/container/app-image.html)
provides in-depth information about content, building, tuning and so on for this image. You should also consult
the [Dataverse Container Guide - Base Image](https://guides.dataverse.org/en/latest/container/base-image.html) page
for more details on tunable settings, locations, etc.

**Where to get help and ask questions:**

IQSS will not offer support on how to deploy or run it. Please reach out to the community for help on using it.

> You can join the Community Chat on Matrix at https://chat.dataverse.org or the Community Slack at
> https://dataversecommunity.slack.com to ask for help and guidance.

Reviewer: ```suggestion
You are welcome to ask questions at https://chat.dataverse.org and https://groups.google.com/g/dataverse-community
```

Author: Good catch, done.
